andchir/omnipay-bundle/1.0/config/packages/omnipay.yaml

```diff
@@ -0,0 +1,78 @@
+omnipay:
+    success_url: '/profile/history_orders'
+    fail_url: '/'
+    return_url: '/omnipay_return'
+    notify_url: '/omnipay_notify'
+    cancel_url: '/omnipay_cancel'
+    data_keys:
+        paymentId: ['orderNumber']
+        customerEmail: ['customerNumber']
+    gateways:
+        PayPal_Express:
+            parameters:
+                username: xxxxxxxxxxxxx
+                password: xxxxxxxxxxxxxxxxxx
```

Comment:
These should be in environment variables.

Reply:
Sorry, I don't understand what variables you mean. Why is this config is normal, but I can not do that? Like here https://github.com/symfony/recipes-contrib/blob/master/liip/imagine-bundle/1.8/config/packages/liip_imagine.yaml

Reply:
I would not pass the liip recipe today. That file does not bring any value. 

https://github.com/symfony/recipes-contrib/pull/653